web-local/start.sh

```diff
@@ -1,21 +1,68 @@
 #!/bin/bash
 
-#
-# Starts the following servers locally:
-# - EmoDB (ports 8080, 8081)
-# - Cassandra (port 9160)
-# - ZooKeeper (port 2181)
-#
-# The first time this is run, Cassandra will be initialized with a default
-# schema and an empty # data set.  Data will be stored in "target/cassandra".
-# On subsequent runs where "target/cassandra" already exists, the Cassandra
-# schema and data will not be modified.
-#
-# Once the server is running you can access the Cassandra command line
-# interface using the following commands:
-#
-#   cd target/cassandra/bin
-#   java -jar cassandra-cli.jar
-#
-
-mvn verify -P init-cassandra,start-emodb
+CONFIG_FILE="config-local.yaml"
+DDL_FILE="config-ddl-local.yaml"
+
+function print_usage_and_exit {
+    cat <<EOF
+    $(basename $0) - Start EmoDB locally.
+
+
+     Starts the following servers locally (using ${DDL_FILE} and ${CONFIG_FILE}):
+     - EmoDB (ports 8080, 8081)
+     - Cassandra (port 9160)
+     - ZooKeeper (port 2181)
+
+     The first time this is run, Cassandra will be initialized with a default
+     schema and an empty # data set.  Data will be stored in "target/cassandra".
+     On subsequent runs where "target/cassandra" already exists, the Cassandra
+     schema and data will not be modified.
+
+     Once the server is running you can access the Cassandra command line
+     interface using the following commands:
+
+       cd target/cassandra/bin
+       java -jar cassandra-cli.jar
+
+     Options:
+
+        --ddl-file       Which ddl file to use [Default: ${DDL_FILE}]
+        --config-file    Which config file to use [Default: ${CONFIG_FILE}]
```

Comment:
Minor: These files are relative to `emodb/web-local`.  Maybe add a comment to that effect?